javagen/src/main/java/com/azure/autorest/model/clientmodel/ClassType.java

```diff
@@ -179,6 +179,9 @@ public final String defaultValueExpression(String sourceExpression) {
     }
 
     public final IType getClientType() {
+        if (JavaSettings.getInstance().isLowLevelClient())  {
+            return this;
+        }
```

Comment:
This seems not very understandable to the design of `IType`.

Would it better to have `SchemaMapper` just output e.g. `Long` instead of `UnixTimeLong` in case of LLC?
@jianghaolu

Reply:
Yes I agree - better to handle this in the mappers

Reply:
Fixed